### DISCUSSION TITLE
Any chance for feedback from garage door to Sonoff 2ch?

### DISCUSSION THREAD

**Discussion by kpreobra** [RavenSystem/esp-homekit-devices#1831] [Q&A]
I have sonoff 2ch installed to open garage gates via 1ch and "pedestrian opening" via 2ch.
Is there any possibility to get sonoff to know if garage door was opened or closed from remote control, not from sonoff? Remote control sensor attached in parallel with Sonoff. Control panel of gates turning on +5v when gates are not closed, is it possible to use it?

**Comment by mariusvasss** [comment]
{"c":{"l":13, "m":60,"b":[{"g":0,"t":5}],"0":{"r":[{"g":12}]}} 
,"a":[{"t":40,"d":18,"f3":[{"g":9,"t":0}],"f4":[{"g":9}],"0":{"r":[{"g":12,"v":1,"i":0.5}]},"1":{"
a":0},"2":{"r":[{"g":12,"i":1.5},{"g":12,"v":1,"i":0.5},{"g":12,"v":1,"i":2}]},"3":{"a":2}},"es":
[{"t":40,"d":18,"f3":[{"g":10,"t":0}],"f4":[{"g":10}],"0":{"r":[{"g":5,"v":1,"i":0.5}]},"1":{"a":
0},"2":{"r":[{"g":5,"i":1.5},{"g":5,"v":1,"i":0.5},{"g":5,"v":1,"i":2}]},"3":{"a":2}}]}

Dual Garage Doors with 2 magnetic sensors, change the time for each and the Sensor to use one Sensor.

**Reply by mariusvasss** [reply]
The JSON is for sonoff dual.

**Reply by kpreobra** [reply]
Thanks a lot! Which pins must be used to connect sensor?

**Reply by kpreobra** [reply]
Is there anybody who know the answer?
